.flake8

```diff
@@ -22,4 +22,4 @@ select = C,D,E,F,W,B,B950
 # operators.  It no longer agrees with PEP8.  See, for example, here:
 # https://github.com/ambv/black/issues/21. Guido agrees here:
 # https://github.com/python/peps/commit/c59c4376ad233a62ca4b3a6060c81368bd21e85b.
-ignore = E501,W503
+ignore = E501,W503,F401
```

Comment:
You should remove `F401` from this list- you have globally disabled that Flake8 check for all of your files, which we don't want to do.

Instead, you need to tell Flake8 to ignore that one particular import in `src/pe_reports/__init__.py` [like we do in the skeleton](https://github.com/cisagov/skeleton-python-library/blob/b85e1b85ce7816b8961c0afb2fb9b843ba6e96a5/src/example/__init__.py#L2-L6) by adding `# noqa: F401` to the appropriate line: https://github.com/cisagov/pe-reports/blob/78d906ac241d76e366312eb90548c666321e3b31/src/pe_reports/__init__.py#L6

Reply:
Reverted to original state.

Reply:
Resolved in bbf7aa7